glslc/src/file_compiler.h

```diff
@@ -64,12 +64,22 @@ class FileCompiler {
     output_file_name_ = file;
   }
 
-  // Sets the flag to indicate emitting SPIRV binary code as a C-style
-  // initializer list for a 32-bit unsigned integer array.
-  void SetEmitSpirvBinaryAsCInitializerList() {
-    emit_binary_as_c_init_list_ = true;
+  // Sets the flag to indicate emitting SPIR-V binary code directly.
+  void SetEmitSpirvBinaryAsBinary() {
```

Comment:
Hi @dekimir, do you have a suggestion for this? To have the enum value as a parameter, we will need to expose the enum type to main.cc. Do you think that is a better design?